### PR TITLE
1.7 release notes

### PR DESCRIPTION
**Pull request by MarianRaphael**
## Description
1.7 Release notes highlights:

- Presenting Remote Node-RED Editor Access for Devices
- Device Status Visualisation

## Tasks
- [x] Update YouTube Links
- [x] Add Highlight Image
- [x] Add New Contributors

### CODE REVIEW

File: src/blog/2023/05/flowforge-1-7-released.md [hunk 1]
```diff
@@ -0,0 +1,105 @@
+---
+title: FlowForge 1.7 Now Available
+subtitle: FlowForge now supports access to the Node-RED Editor on devices
+description: FlowForge now supports access to the Node-RED Editor on devices
+date: 2023-05-11 
+authors: ["marian-demme"]
+# image: /blog/2023/04/images/release-1.6.0.png
+---
+
+The new FlowForge 1.7 adds new support for accesing the Node-RED Editor on Devices via FlowForge
```
Comment (robmarcer) [inline src/blog/2023/05/flowforge-1-7-released.md:10]:
```suggestion
FlowForge 1.7 adds new support for accessing the Node-RED Editor on Devices via FlowForge
```

File: src/blog/2023/05/flowforge-1-7-released.md [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+## Try it out
+
+We're confident you can have self managed FlowForge running locally in under 30 minutes.
+You can install our [local build](https://flowforge.com/docs/install/local/), use [Docker](https://flowforge.com/docs/install/docker/), or [Kubernetes](https://flowforge.com/docs/install/kubernetes/).
+
+If you'd rather use our hosted offering: [Get started for free](https://app.flowforge.com/account/create) on FlowForge Cloud.
+
+## Upgrading FlowForge
+
+[FlowForge Cloud](https://app.flowforge.com) is already running 1.6.
```
Comment (robmarcer) [inline src/blog/2023/05/flowforge-1-7-released.md:85]:
```suggestion
[FlowForge Cloud](https://app.flowforge.com) is already running 1.7.
```

File: src/blog/2023/05/flowforge-1-7-released.md [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+<iframe width="560" height="315" src="https://youtu.be/dQw4w9WgXcQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Device Status Visualisation
+
+This upgrade simplifies device monitoring, making it even easier to manage your devices effectively. This new feature offers an intuitive, user-friendly method for users to keep an eye on their devices' status and evaluate the health of their team's devices overall. 
+
+<iframe width="560" height="315" src="https://youtu.be/dQw4w9WgXcQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Boosts Node-RED Instance Reliability
```
Comment (robmarcer) [inline src/blog/2023/05/flowforge-1-7-released.md:26]:
```suggestion
## Auto Restart for Hung Node-RED Instances
```

File: src/blog/2023/05/flowforge-1-7-released.md [hunk 4]
```diff
@@ -0,0 +1,105 @@
...
+
+This enhancement ensures a more robust and reliable experience when working with Node-RED flows. The launcher now actively monitors the Node-RED process to detect if it has become unresponsive or hung, in addition to the existing checks for start-up and unexpected process exits. This advancement takes us one step further in enhancing the availability of our Node-RED instances.
+
+[See details on GitHub](https://github.com/flowforge/flowforge-nr-launcher/issues/110)
+
+## Ongoing Topics
+
+### High Availibility
+
+We're pleased to share our progress on enhancing High Availability in FlowForge. Based on customer feedback, our focus is implementing a hot-spare approach for Node-RED applications, providing automatic failover between instances for improved reliability and availibility. We've identified our initial set of tasks and changes to how we'll run Node-RED instance with the k8s environment to create a more robust system. For a detailed update about High Availibility you should definitely checkout our newest [Article](./....) about High Availibility.
```
Comment (robmarcer) [inline src/blog/2023/05/flowforge-1-7-released.md:36]:
```suggestion
We're pleased to share our progress on enhancing High Availability in FlowForge. Based on customer feedback, our focus is implementing a hot-spare approach for Node-RED applications, providing automatic failover between instances for improved reliability and availability. We've identified our initial set of tasks and changes to how we'll run Node-RED instances with the k8s environment to create a more robust system. For a detailed update about High Availability you should definitely checkout our newest [Article](./....) about High Availability.
```

File: src/blog/2023/05/flowforge-1-7-released.md [hunk 5]
```diff
@@ -0,0 +1,105 @@
...
+
+## Boosts Node-RED Instance Reliability
+
+This enhancement ensures a more robust and reliable experience when working with Node-RED flows. The launcher now actively monitors the Node-RED process to detect if it has become unresponsive or hung, in addition to the existing checks for start-up and unexpected process exits. This advancement takes us one step further in enhancing the availability of our Node-RED instances.
+
+[See details on GitHub](https://github.com/flowforge/flowforge-nr-launcher/issues/110)
+
+## Ongoing Topics
+
+### High Availibility
```
Comment (robmarcer) [inline src/blog/2023/05/flowforge-1-7-released.md:34]:
```suggestion
### High Availability
```

**Review by robmarcer — CHANGES_REQUESTED** [review]

File: src/blog/2023/05/flowforge-1-7-released.md [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+
+Enable 'Delete Team' Button [#2031](https://github.com/flowforge/flowforge/issues/2031)
+
+Triggering a Node-RED restart using the action button resulted in two instances of Node-RED running in the container, causing one instance to crash due to port 1880 being already in use. [#2031](https://github.com/flowforge/flowforge/issues/1860)
+
+Error deleting instance with missing subscription [#2080](https://github.com/flowforge/flowforge/issues/2080)
+
+Snapshot Rollback no longer working [#2026](https://github.com/flowforge/flowforge/issues/2026)
+
+Users receiving an unauthorized error when attempting to switch to a team in which they are a membe [#1845](https://github.com/flowforge/flowforge/issues/1845)
```
Comment (robmarcer) [inline src/blog/2023/05/flowforge-1-7-released.md:72]:
```suggestion
Users receiving an unauthorized error when attempting to switch to a team in which they are a member [#1845](https://github.com/flowforge/flowforge/issues/1845)
```

File: src/blog/2023/05/flowforge-1-7-released.md [hunk 7]
```diff
@@ -0,0 +1,105 @@
...
+
+## Presenting Remote Node-RED Editor Access for Devices
+
+We are excited to introduce a new feature that will simplify the process of debugging and developing flows for devices. Our latest feature, "Editing Flows on Devices," allows users to access the editor directly on their device without the need for complex network configurations or firewalls. This feature will significantly improve the user experience, making it easier and more efficient to work with devices.
+
+<iframe width="560" height="315" src="https://youtu.be/dQw4w9WgXcQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Device Status Visualisation
+
+This upgrade simplifies device monitoring, making it even easier to manage your devices effectively. This new feature offers an intuitive, user-friendly method for users to keep an eye on their devices' status and evaluate the health of their team's devices overall.
```
Comment (ZJvandeWeg) [inline src/blog/2023/05/flowforge-1-7-released.md:22]:
```suggestion
This upgrade simplifies device monitoring, making it even easier to manage your devices effectively. This new feature offers an intuitive, user-friendly method for users to keep an eye on their devices' status and evaluate the health of their team's devices overall. Creating an overview of your fleets health however large your fleet might be.
```

File: src/blog/2023/05/flowforge-1-7-released.md [hunk 8]
```diff
@@ -0,0 +1,105 @@
...
+
+This upgrade simplifies device monitoring, making it even easier to manage your devices effectively. This new feature offers an intuitive, user-friendly method for users to keep an eye on their devices' status and evaluate the health of their team's devices overall. 
+
+<iframe width="560" height="315" src="https://youtu.be/dQw4w9WgXcQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+## Auto Restart for Hung Node-RED Instances
+
+This enhancement ensures a more robust and reliable experience when working with Node-RED flows. The launcher now actively monitors the Node-RED process to detect if it has become unresponsive or hung, in addition to the existing checks for start-up and unexpected process exits. This advancement takes us one step further in enhancing the availability of our Node-RED instances.
+
+[See details on GitHub](https://github.com/flowforge/flowforge-nr-launcher/issues/110)
```
Comment (ZJvandeWeg) [inline src/blog/2023/05/flowforge-1-7-released.md:30]:
```suggestion
[See also our documentation](/docs/.....)
```

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

File: src/blog/2023/05/flowforge-1-7-released.md [hunk 9]
```diff
@@ -0,0 +1,108 @@
+---
+title: FlowForge 1.7 Now Available
```
Comment (ZJvandeWeg) [inline src/blog/2023/05/flowforge-1-7-released.md:2]:
```suggestion
title: FlowForge 1.7 Now Available with Remote Node-RED Editor Access
```

File: src/blog/2023/05/flowforge-1-7-released.md [hunk 10]
```diff
@@ -0,0 +1,108 @@
...
+subtitle: FlowForge now supports access to the Node-RED Editor on devices
```
Comment (ZJvandeWeg) [inline src/blog/2023/05/flowforge-1-7-released.md:3]:
```suggestion
subtitle: Further improving fleet management and maintenance of remote Node-RED instances
```

What about this for a title / subtitle?

File: src/blog/2023/05/flowforge-1-7-released.md [hunk 11]
```diff
@@ -0,0 +1,108 @@
...
+description: FlowForge now supports access to the Node-RED Editor on devices
+date: 2023-05-11 
+authors: ["marian-demme"]
+image: /blog/2023/05/images/release-1.7.0.png
+---
+
+FlowForge 1.7 adds new support for accessing the Node-RED Editor on Devices via FlowForge
```
Comment (ZJvandeWeg) [inline src/blog/2023/05/flowforge-1-7-released.md:10]:
```suggestion
FlowForge 1.7 adds new support for accessing the Node-RED Editor on Devices via FlowForge.
```

**Review by ZJvandeWeg — CHANGES_REQUESTED** [review]

**Comment by hardillb** [comment]
New contributors

**Review by robmarcer — CHANGES_REQUESTED** [review]

**Review by knolleary — APPROVED** [review]

**Pull request merged**